planning/costmap_generator/launch/costmap_generator.launch.xml

```diff
@@ -1,3 +1,15 @@
+<!--
+  Copyright 2021-2023 UCI SORA Lab, the Autoware Foundation
```

Comment:
Thanks for adding the copyright notice to the launch file.
However, since some parts of the code comes from different author (prehaps TIER IV), we might want to have the original author mentioned as well in the notice.

Reply:
For the policy on copyright notice, let's discuss in today's Open AD Kit meeting.

Reply:
Conclusion from the WG discussion is posted in this issue
https://github.com/autowarefoundation/autoware.universe/issues/2652

Reply:
The copyright notice has been modified.